src/rules/de.toml

```diff
@@ -24,6 +24,16 @@ replacements = [
   ["evtl.", "eventuell"],
   ["bzgl.", "bezüglich"],
   ["Nr.", "Nummer"],
+  ["rd.", "rund"], 
+  ["hl.", "heilig"],
```

Comment:
Just came to my mind, I don't think this one will work nicely. I generally think that adjectives won't work nicely due to the different cases.

"Die hl. drei Könige" -> heiligen
"Die hl. Maria" -> heilige